content/docs/2.0/concepts/scaling-jobs.md

```diff
@@ -106,7 +109,45 @@ The max number of pods that is created within a single polling period. If there
 * **Target Average Value:** The number of messages that will be consumed on a job. It is defined on the scaler side. e.g. `queueLength` on `Azure Storage Queue` scaler.
 * **Running Job Count:** How many jobs are running.
 * **Number of the Scale:** The number of the job that is created.
+```yaml
+  scalingStrategy: "accurate"                 # Optional. Default: default Which ScalingStrategy to use. 
+```
+
+Select a Scaling Strategy. Possible values are `default`, `custom`, or `accurate`. The default value is `default`.
+
+### default
```

Comment:
Formatting nit:
I'd change this from `### default` to `** default **` (the same applies to the custom and accurate section) or to change this to bulleted list or something like that, I wouldn't use a header for this parts, because it is being rendered on the same level as `Details` section, which is not correct.

Reply:
Done